### PR TITLE
Add check broken links job for managed-notifications 

### PR DESCRIPTION
**Pull request by YiqinZhang**
- Create CI jobs to detect broken links in the openshift-managed-notifications repo during PR checks and on a weekly basis.
- If any broken links are found, send an email to sd-docs@redhat.com with the list of broken links.

[SDCICD-1162](https://issues.redhat.com/browse/SDCICD-1162)

### CODE REVIEW

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

File: ci-operator/config/openshift/managed-notifications/openshift-managed-notifications-master.yaml [hunk 1]
```diff
@@ -15,6 +15,58 @@ tests:
...
+    done
+    if [ -s "$TEMP_FILE" ]; then
+        echo "Broken links detected:" | tee -a "$LOG_FILE"
+        cat "$TEMP_FILE" | tee -a "$LOG_FILE"
+        exit 1
+    else
+        echo "No broken links detected." | tee -a "$LOG_FILE"
+        exit 0
+    fi
+    rm -rf temp_repo
```
Comment (afreiberger) [inline ci-operator/config/openshift/managed-notifications/openshift-managed-notifications-master.yaml:66]: With the addition of exit 0 in success "else" above will skip this command every run.  Was this intended?

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-managed-notifications-master-checklinks-weekly

**Pull request merged**